engine/src/main/java/org/terasology/world/block/family/BlockSections.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.world.block.family;
+
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Describes the sections that a block can use such as top, left, right, bottom
+ * but other sections can be used.
+ * <p>
+ *     Each section will override the existing values on the base block when that family section is used. look in
```

Comment:
> Each section will override the existing values on the base block when that family section is used

What do you want to express by this? Does it mean that there is only one _active_ isntance of `BlockSections`  per family (or per block?)